modules/carousel/Carousel.tsx

```diff
@@ -222,79 +248,106 @@ const Carousel: React.FC<ICarouselProps & WithStylesProps<Theme, ICarouselStyles
     addEvent("mouseleave", scrollableElementRef?.current, () =>
       onMouseUp(scrollableElementRef?.current)
     );
-  }, [scrollableElementRef]);
+  }, [blockScroll, scrollableElementRef]);
 
   const calculateChildrenSize = useCallback<() => number>(() => {
```

Comment:
This method is being called every time there is a scroll. Is that necessary? Couldn't we store the result of this method in a `useMemo` (and update that value whenever necessary, but not on each scroll)?

Reply:
this is wrapped with an useCallback so this won't be called on each scroll, only the memoized value